bobtemplates/odoo/hooks.py

```diff
@@ -147,29 +167,61 @@ def post_render_model(configurator):
 
 
 #
-# addon hooks
+# readme hooks
 #
 
 
-def pre_render_addon(configurator):
+def pre_render_readme(configurator):
     variables = configurator.variables
-    variables["odoo.version"] = int(variables["addon.version"].split(".")[0])
     variables["addon.name_camelwords"] = _underscored_to_camelwords(
         variables["addon.name"]
     )
 
 
+def post_render_readme(configurator, addon_name=""):
+    variables = configurator.variables
+    addon_name = addon_name + "/" if addon_name else addon_name
+    oca = (
+        variables["addon.oca"]
+        if isinstance(variables["addon.oca"], bool)
+        else to_boolean(False, False, variables["addon.oca"])
+    )
+    if oca:
+        _get_oca_readme_fragments(configurator, addon_name)
+        _delete_file(configurator, addon_name + "README.rst.acsone")
+    else:
+        _rm_suffix(".acsone", configurator, addon_name + "README.rst.acsone")
+        # Remove readme directory
+        path = os.path.join(configurator.target_directory, addon_name + "readme")
+        os.removedirs(path)
+
+
+#
+# addon hooks
+#
+
+
+def pre_render_addon(configurator):
+    pre_render_readme(configurator)
+    variables = configurator.variables
+    variables["odoo.version"] = int(variables["addon.version"].split(".")[0])
+
+
 def post_render_addon(configurator):
     variables = configurator.variables
-    if variables["addon.oca"]:
-        _rm_suffix(".oca", configurator, variables["addon.name"] + "/README.rst.oca")
+    post_render_readme(configurator, addon_name=variables["addon.name"])
+    oca = (
+        variables["addon.oca"]
+        if isinstance(variables["addon.oca"], bool)
+        else to_boolean(False, False, variables["addon.oca"])
```

Comment:
It's seems to_boolean is not executed on variabes. It's why a added this part of code